docs/pages/heading.js

```diff
@@ -399,9 +404,10 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         </MainSection.Subsection>
         <MainSection.Subsection
           title="Accessibility level"
-          description="For accessibility purposes, we allow you to override the heading level.
+          description={`
+For accessibility purposes, we allow you to override the heading level.
 
-        We should have one level 1 per page &amp; levels should be appropriately nested. E.g. level 1 followed by level 2 &amp; level 2 followed by level 2 or level 3. We also allow headings without an accessibility level."
+We should have one level 1 per page &amp; levels should be appropriately nested. E.g. level 1 followed by level 2 &amp; level 2 followed by level 2 or level 3. We also allow headings without an accessibility level. We recommend against using \`accessibilityLevel="none"\`.`}
```

Comment:
These two sentences read a bit awkward together. Consider combining to something like "For certain specific situations, it is possible to use Heading with an accessibility level; however, we recommend against using this if possible."

Reply:
Done!